app/models/application.rb

```diff
@@ -149,4 +136,12 @@ def sponsor_pick?
       value.to_s != '0' ? flags.concat([flag.to_s]).uniq : flags.delete(flag.to_s)
     end
   end
+
+  def project1
+    Project.find_by(id: application_data['project1_id'])
```

Comment:
Ok, just to quickly clearify my understanding of DTOs. The way I usually use them, they are POROs taking KWARGS and then internally just delegate to the underlying objects. So they are basically just a wrapper around objects that are passed around in business-logic as this one pack which then can be easily stubbed in specs.

So they usually just have a few private `attr_reader` and some large lists of `def_delegators` 😉 , no *"real"* methods.

But yeah, curios of what you have in mind here.